### DISCUSSION TITLE
changing the subscriber's IP address

### DISCUSSION THREAD

**Discussion by yniino** [chime-frb-open-data/community#96] [Q&A]
Hi CHIME/FRB team,

I have recently moved to a different institute, and my VOEvent subscriber machine has a new IP address. Please let me know how to change the registered IP address. (I also receive email alerts, but don't need to change email address.)

Thanks, 
Yuu

**Comment by tabbott36** [comment]
Hi Yuu, 

Thank you for your question! There is currently no way to change your existing IP subscription, so you will have to submit a new subscription request here: https://www.chime-frb.ca/voevents. 

Apologies for the inconvenience, please let me know if you have any more questions!

- Thomas

**Reply by yniino** [reply]
Hi Thomas, 

Thank you for your prompt reply. 

I'm subscribing to both email and VOEvent alerts. Let me make sure that my email subscription will not be deleted or duplicated by the new subscription. To modify only the registered IP address keeping the email subscription unchanged, is submitting a new subscription only for the VOEvent subscription fine? or should I make another VOEvent+email subscription? 

Thanks,
Yuu

**Reply by tabbott36** [reply]
Hi Yuu, 

Yes, submitting a new subscription request will be fine - you're email subscription will be unaffected.  The new subscription request is completely independent from the existing subscription.

All the best, 

-Thomas

**Reply by yniino** [reply]
Hi Thomas, 

Thank you for your advice. I have submitted the new subscription request. 

Thanks, 
Yuu
